aldryn_redirects/admin.py

```diff
@@ -58,12 +58,12 @@ def export_view(self, request):
         filename = timezone.now().date().strftime(self.export_filename)
         redirects = self.get_queryset(request).prefetch_related('translations')
 
-        for redirect in redirects:
+        for r in redirects:
```

Comment:
what is the reason for this change?

Reply:
Avoid shadowing of global 'redirect' var (import)